external_libraries/csv-parser/csv.hpp

```diff
@@ -4395,6 +4395,7 @@ HEDLEY_DIAGNOSTIC_POP
 #  if \
      (__cplusplus >= 201703L) || \
      ((__cplusplus >= 201103L) && HEDLEY_HAS_CPP_ATTRIBUTE(fallthrough))
+#pragma clang diagnostic ignored "-Wc++1z-extensions"
```

Comment:
hm ok
Strange that they enable C++17 features ...

in any case, please also disable the warning removal so that it doesn't leak into other files

Reply:
Is a pragma, I think it only affects to this

Reply:
it does: https://clang.llvm.org/docs/UsersManual.html#id28

Reply:
sorry thats the correct link: https://clang.llvm.org/docs/UsersManual.html#controlling-diagnostics-via-pragmas